plugins/maciVoting/services/coordinator/utils/types.ts

```diff
@@ -0,0 +1,349 @@
+   * The amount of initial voice credits to deploy
+   */
+  amount: number;
+}
+
+/**
+ * IEASPolicyArgs represents the arguments for deploying an EAS policy
+ */
+export interface IEASPolicyArgs {
+  /**
+   * The address of the EAS contract
+   */
+  easAddress: string;
+
+  /**
+   * The attestation schema to be used
+   */
+  schema: string;
+
+  /**
+   * The trusted attester
+   */
+  attester: string;
+}
+
+/**
+ * IZupassPolicyArgs represents the arguments for deploying a Zupass policy
+ */
+export interface IZupassPolicyArgs {
+  /**
+   * The first signer
+   */
+  signer1: string;
+
+  /**
+   * The second signer
+   */
+  signer2: string;
+
+  /**
+   * The event ID
+   */
+  eventId: string;
+
+  /**
+   * The Zupass verifier address
+   */
+  zupassVerifier: string;
+}
+
+/**
+ * IHatsPolicyArgs represents the arguments for deploying a Hats policy
+ */
+export interface IHatsPolicyArgs {
+  /**
+   * The hats protocol address
+   */
+  hatsProtocolAddress: string;
+
+  /**
+   * The criterion hats
+   */
+  critrionHats: string[];
+}
+
+/**
+ * ISemaphorePolicyArgs represents the arguments for deploying a semaphore policy
+ */
+export interface ISemaphorePolicyArgs {
+  /**
+   * The semaphore contract address
+   */
+  semaphoreContract: string;
+
+  /**
+   * The group ID
+   */
+  groupId: string;
+}
+
+/**
+ * IMerkleProofPolicyArgs represents the arguments for deploying a merkle proof policy
+ */
+export interface IMerkleProofPolicyArgs {
+  /**
+   * The merkle proof root
+   */
+  root: string;
+}
+
+/**
+ * ITokenPolicyArgs represents the arguments for deploying a sign up policy
+ */
+export interface ITokenPolicyArgs {
+  /**
+   * The token address
+   */
+  token: string;
+}
+
+/**
+ * IAnonAadhaarPolicyArgs represents the arguments for deploying an Anon Aadhaar policy
+ */
+export interface IAnonAadhaarPolicyArgs {
+  /**
+   * The Anon Aadhaar verifier address
+   */
+  verifier: string;
+
+  /**
+   * The nullifier seed
+   */
+  nullifierSeed: string;
+}
+
+/**
+ * IGitcoinPassportPolicyArgs represents the arguments for deploying a gitcoin passport policy
+ */
+export interface IGitcoinPassportPolicyArgs {
+  /**
+   * The decoder address
+   */
+  decoderAddress: string;
+
+  /**
+   * The passing score
+   */
+  passingScore: string;
+}
+
+/**
+ * IERC20VotesPolicyArgs represents the arguments for deploying an ERC20 votes policy
+ */
+export interface IERC20VotesPolicyArgs {
+  /**
+   * The token address
+   */
+  token: string;
+
+  /**
+   * The threshold
+   */
+  threshold: bigint | string;
+
+  /**
+   * The snapshot block
+   */
+  snapshotBlock: bigint | string;
+}
+
+/**
+ * IERC20PolicyArgs represents the arguments for deploying an ERC20 policy
+ */
+export interface IERC20PolicyArgs {
+  /**
+   * The token address
+   */
+  token: string;
+
+  /**
+   * The threshold
+   */
+  threshold: string;
+}
+
+/**
+ * IPolicyArgs represents the arguments for deploying a policy
+ */
+export type IPolicyArgs =
```

Comment:
The policy type is only `IERC20VotesPolicyArgs` so I think we can just use that type (not even defined IPolicyArgs but use `IERC20VotesPolicyArgs` directly in the `policy: { type:  IERC20VotesPolicyArgs }`